test/CMakeLists.txt

```diff
@@ -650,10 +650,18 @@ if(${MIOPEN_TEST_WITH_MIOPENDRIVER})
         # Issue #1697: this is large test which has to run in serial and not enabled on gfx900/gfx906
         COMMAND MIOPEN_DRIVER_USE_GPU_REFERENCE=1 $<TARGET_FILE:MIOpenDriver> ${MIOPENDRIVER_MODE_CONV} -W 5078 -H 4903 -c 24 -n 5 -k 1 --fil_w 3 --fil_h 3 --pad_w 6 --pad_h 4 -F 1
     )
+
+    add_custom_test(test_miopendriver_conv3d_half_gfx9 SKIP_UNLESS_ALL GFX900_DISABLED GFX906_DISABLED GFX908_DISABLED GFX90A_ENABLED GFX103X_DISABLED HALF_ENABLED
```

Comment:
[Quality] AFAICS the test is enabled only for MI200, for both `float` and `half` types. Using it with `float` datatype is not needed, Please add `FLOAT_DISABLED`.

[Quality] `GFX103X_DISABLED` is not needed because testing for this target is disabled by defaut. Please remove this attribute.

[Quality] Please rename the test to `test_miopendriver_half_mi200`. This would indicate that the test will be executed only for HALF datatype and only on MI200, and suggests the developer to add here any other tests that require MIOpenDriver and should be run only for FP16 on MI200. `Note that conv3d` in the test name is superfluous (because other, non-conv3d related tests could be added here), `gfx9` is ambiguous.